test/fixtures/dummy-ember-try-config-with-npm-scenarios.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
       name: 'test1',
       bower: {
         dependencies: {
-          ember: '2.18.0'
+          'ember-source': '2.18.0'
```

Comment:
this seems wrong. on bower this was not called ember-source yet.

Reply:
yes you're completely right 🙈 I was trying to fix this and I must have overlooked that it was bower